sledgehammer/start-up.sh

```diff
@@ -103,7 +103,7 @@ if ! suse_ver 12; then
 else
     if [ "$BOOTDEV" != "eth0" ]; then
         mv /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-$BOOTDEV
-        wicked ifup --timeout ${WAIT_FOR_INTERFACES:-60} $BOOTDEV
+        wicked ifup --timeout ${WAIT_FOR_INTERFACES:-120} $BOOTDEV
```

Comment:
@nicolasbock yes. the issue is that in the hardware environment it takes between 59 and 63s  to get network. so 60 was at the border and it was flakey.